app/root.tsx

```diff
@@ -238,10 +238,11 @@ export function Layout({ children }: { children?: ReactNode }) {
         <DevBanner />
         <Header />
         <NewsBanner>
-          New Announcement Feature, Code of Conduct, Circular Revisions. See{' '}
+          Introducing Einstein Probe Notices, Astro Flavored Markdown Circulars,
+          and Notices Schema v4.0.0. See{' '}
```

Comment:
This didn't quite fit in one line. Trim a few words to make it fit...

```suggestion
          Introducing Einstein Probe, Astro Flavored Markdown, and Notices
          Schema v4.0.0. See{' '}
```

# Before

<img width="1174" alt="Screenshot 2024-05-30 at 14 45 22" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/728407/11bee6cd-ed05-4803-93e1-db84730b980f">

# After

<img width="1174" alt="Screenshot 2024-05-30 at 14 46 49" src="https://github.com/nasa-gcn/gcn.nasa.gov/assets/728407/76249502-1658-4f6c-a9c9-e1a35597d861">